internal/pgbackrest/reconcile.go

```diff
@@ -111,17 +112,9 @@ func AddConfigToInstancePod(
 	// volumes stay valid and Kubernetes propagates their contents to those pods.
 	secret := corev1.VolumeProjection{Secret: &corev1.SecretProjection{}}
 	secret.Secret.Name = naming.PGBackRestSecret(cluster).Name
+	secret.Secret.Items = append(secret.Secret.Items, clientCertificates()...)
```

Comment:
Why do we need this here?

Reply:
We used to only create the pgbackrest server in the instance if a repo host was present. Now we always create the pgbackrest server in the instance (assuming backups are enabled of course).  The server not only needs a server cert and key (which are added in the `AddServerToInstancePod` function), but also requires the client CA cert which is provided via the `clientCertificates` function... So it's a little confusing because the installation of the necessary pieces for the `server` occur in both `AddServerToInstancePod` and `AddConfigToInstancePod`... It probably could be refactored so that the `server` stuff is fully handled in the former function...